src/test/java/com/wikia/webdriver/testcases/logintests/ForcedLoginTests.java

```diff
@@ -57,27 +58,41 @@ public void anonCanLogInViaAuthModalWhenAddingVideo() {
   }
 
   @Test(groups = "ForcedLogin_anonCanLogInViaUserLoginPage")
-  @RelatedIssue(issueID = "SOC-2669", comment = "fails after product change, SOC team on it")
   public void anonCanLogInViaUserLoginPage() {
     WikiBasePageObject base = new WikiBasePageObject();
     base.openSpecialUpload(wikiURL);
     base.verifyLoginReguiredMessage();
     SpecialUserLoginPageObject special = base.clickLoginOnSpecialPage();
-    special.login(credentials.userName10, credentials.password10);
-    special.verifyUserLoggedIn(credentials.userName10);
+
+    SignInPage signInPage = new SignInPage(driver);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "signInPage" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)